node-src/lib/setExitCode.ts

```diff
@@ -25,6 +25,9 @@ export const exitCodes = {
   STORYBOOK_START_FAILED: 22,
   STORYBOOK_BROKEN: 23,
 
+  // E2E errors
+  E2E_BUILD_FAILED: 51,
+
```

Comment:
Should we add this to the CLI [exit codes](https://www.chromatic.com/docs/cli/#exit-codes) in the documentation?

Reply:
@skitterm sure, I can open a PR for that.